pnpm-lock.yaml

```diff
@@ -5001,13 +4934,13 @@ packages:
   '@tediousjs/connection-string@0.3.0':
     resolution: {integrity: sha512-d/keJiNKfpHo+GmSB8QcsAwBx8h+V1UbdozA5TD+eSLXprNY53JAYub47J9evsSKWDdNG5uVj0FiMozLKuzowQ==}
 
-  '@theguild/remark-mermaid@0.1.2':
-    resolution: {integrity: sha512-CB1n996M3JqR/9yAOchdOtfmtvDASCUlg/M4wJ/dmZmEpLNlBko8V31Nszo5Uz9+xaO6NrYcjEXA9i/PxzI1xQ==}
```

Comment:
Hi everyone, this error seems related to `@theguild/remark@0.1.2`, and updating the lockfile to install `@theguild/remark@0.1.3` might resolve the issue. You can find the release [here](https://github.com/the-guild-org/docs/releases/tag/release-1727376017280).